### ISSUE TITLE
adopt-openjdk install failure: missing checksum_link causes install failure

### ISSUE THREAD

**Issue by adyang** [halcyon/asdf-java#72]
For newer versions of adopt-openjdk, it seems that "checksum_link" is no longer present under the "packages" field which will cause installation to fail with error "curl: Remote file name has no length!".

Command to reproduce:
```console
$ asdf install java 'adopt-openjdk-11.0.6+10'
######################################################################## 100.0%
OpenJDK11U-jdk_x64_mac_hotspot_11.0.6_10.tar.gz
curl: Remote file name has no length!
curl: try 'curl --help' or 'curl --manual' for more information
```

**Comment by rubencaro** [comment]
Isn't the same as https://github.com/halcyon/asdf-java/issues/71? Can the same PR https://github.com/halcyon/asdf-java/pull/74 fix it too?

**Comment by adyang** [comment]
> Isn't the same as #71? Can the same PR #74 fix it too?
> 
> And viceversa 😁.

Hi yes, think it is the same issue and the PR would fix it too!
Did not realise that it was the same issue as the title was different.

**Comment by android10** [comment]
Any updates on this one?

**Comment by halcyon** [comment]
I appreciate the good work from @adyang. I'm in progress on some ideas that I think may be able to clean up our code base and resolve this issue at the same time. You can expect a resolution from me within the next day or so. Thank you again for all your help and bringing this problem to my attention.

**Comment by rubencaro** [comment]
@halcyon awesome job. Thanks!

**Comment by adyang** [comment]
Fixed in #77
